spec/javascripts/components/notes/note_form_spec.js

```diff
@@ -0,0 +1,44 @@
+import jasmineEnzyme from 'jasmine-enzyme';
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import NoteForm from 'components/notes/NoteForm';
+import Note from 'models/note.js';
+
+const note = new Note({note: ''});
+
+describe('<NoteForm />', function() {
+
+  beforeEach(function() {
+    jasmineEnzyme();
+    sinon.stub(I18n, 't');
+    sinon.stub(window.md, 'makeHtml');
+    sinon.stub(note, 'save');
+  });
+
+  afterEach(function() {
+    I18n.t.restore();
+    window.md.makeHtml.restore();
+    note.save.restore();
+  });
+
+  it("should be able to save a note", function() {
+    const wrapper = shallow(
+      <NoteForm note={note} />
+    );
+    wrapper.find('.add-note').simulate('click');
+    expect(note.save).toHaveBeenCalled();
+  });
+
+  it("should update its own value", function() {
+    const wrapper = shallow( <NoteForm note={note} /> );
+    wrapper.find('textarea').simulate('change', {
+        target: {
+            name: 'note',
```

Comment:
Looks like you're using 4-spaces indentation here, we're using 2-spaces indentation in this project.